ci/cpp-python-msvc-build.bat

```diff
@@ -133,6 +133,8 @@ popd
 
 pushd python
 
+pip install pickle5
```

Comment:
This install call is redundant with the once below.

Reply:
Actually, no, the second one is in a distinct virtualenv where we install the wheel we just built.

Reply:
Ah, I got confused by the free-standing `pip install` here with no other packages. This is then just because we have no conda package for it yet?

Reply:
Probably, but it's very quick to compile anyway and there are no non-Python dependencies.

Reply:
FWIW we did add a package to `conda-forge`. Though it's true this is quite fast to build.

ref: https://github.com/conda-forge/pickle5-feedstock